### ISSUE TITLE
Audit #1 – General Overview when logged in – Visual labels of buttons and their accessible names differ in wording and human language

### ISSUE THREAD

**Issue by marcus-herrmann** [owncloud/web#4340]
Connected to Audit 1, see #4300 
1.2 General Overview when logged in, URL: https://ocis-a11y.owncloud.works/#/files/list

# Issue
* Name of clickable "Table headers" above file list: Visual label ("Name") and accessible name ("Sort files by name") differ, even in human language. Although in this case, "name" is in "Sort files by name", so dogmatically no violation
* Top bar button regarding profile: Visual label "Albert Einstein" (or the full name of the user, repectively), but accessible name "User Menu" – this is a mismatch and therefore a violation

# Remediation
Make sure that the visual label ("button text") of a control is part of its accessible name (for example provided by aria-label). Example for the second issue: Visual "Albert Einstein", aria-label="Profil bearbeiten: Albert Einstein". https://www.w3.org/WAI/WCAG21/Techniques/general/G208.html, example 3.

**Comment by pascalwengerter** [comment]
Closing this since #5018 is merged and we're waiting for feedback (and will open new tickets with the requested changes from the `a11y` audit)
